### PR TITLE
Convert MSI to WIX where appropriate

### PR DESCRIPTION
**Pull request by Trenly**
@OfficialEsco @jedieaston This should automatically differentiate between WIX and MSI.

If an application is currently listed as MSI, it will check if it is MSI or WIX when performing a quick upgrade or an auto-upgrade, and change it if necessary

### CODE REVIEW

**Comment by Trenly** [comment]
Note to self - This function will allow for some fields like copyright to be filled automatically for some installers, of all types

File: Tools/YamlCreate.ps1 [hunk 1]
```diff
@@ -427,7 +469,13 @@ Function Get-PathInstallerType {
     )
 
     if ($Path -match '\.msix(bundle){0,1}$') { return 'msix' }
-    if ($Path -match '\.msi$') { return 'msi' }
+    if ($Path -match '\.msi$') {
+        $ObjectMetadata = Get-ItemMetadata $Path
+        if ($ObjectMetadata.Keys -contains 'ProgramName') {
+            if ($ObjectMetadata.ProgramName -match 'Windows Installer XML') { return 'wix' }
```
Comment (OfficialEsco) [inline Tools/YamlCreate.ps1:475]: This field can be different, check PRTG for ref, AFAIK pure MSI is always ‘Windows Installer’

Comment (Trenly) [reply Tools/YamlCreate.ps1:475]: Ahh, will update

**Review by OfficialEsco — APPROVED** [review]

**Comment by Trenly** [comment]
@jedieaston This look accurate?

**Review by jedieaston — APPROVED** [review]
:shipit:

**Pull request merged**